src/htmlminifier.js

```diff
@@ -265,7 +265,7 @@ function isSrcset(attrName, tag) {
 async function cleanAttributeValue(tag, attrName, attrValue, options, attrs) {
   if (isEventAttribute(attrName, options)) {
     attrValue = trimWhitespace(attrValue).replace(/^javascript:\s*/i, '');
-    return await options.minifyJS(attrValue, true);
+    return options.minifyJS(attrValue, true);
```

Comment:
Yeah, this is implicit. Both behaves same way. I came to know this from an [eslint-rule,](https://eslint.org/docs/rules/no-return-await) this avoids an extra microtask. Since we are not handling any errors here, this should do fine.